setup.py

```diff
@@ -38,11 +35,6 @@ def run(self):
         _sdist.run(self)
 
 
-class install(_install):
```

Comment:
Ah, alright. I'll take that into account when I poke at this and see what I can come up with.

Thanks for the info.